cellrank/kernels/_velocity_kernel.py

```diff
@@ -48,13 +53,15 @@ def __init__(
         backward: bool = False,
         xkey: Optional[str] = "Ms",
         vkey: Optional[str] = "velocity",
+        attr: Optional[Literal["layers", "obsm"]] = "layers",
```

Comment:
Please move this argument below `backward` to maintain the order in the signature.

Reply:
It's just for consistency to maintain the same order in the signature as well as in the docs.